apps/myaccount/package.json

```diff
@@ -34,7 +34,7 @@
         "typecheck": "pnpm run compile"
     },
     "dependencies": {
-        "@asgardeo/auth-react": "^1.1.24",
+        "@asgardeo/auth-react": "^3.1.1",
```

Comment:
Since this is a huge version upgrade, there might be some flows breaking due to this. Hope you have done sufficient smoke testing around the app :)

Reply:
Did some testing as mentioned in https://github.com/wso2/identity-apps/pull/5050#issuecomment-1853633062.